lidarrmetadata/provider.py

```diff
@@ -936,17 +748,26 @@ def __init__(self):
         self._client = mediawikiapi.MediaWikiAPI()
 
     def get_artist_overview(self, url):
```

Comment:
Looks like there's a `NameError: global name 'url' is not defined` here when requesting https://api.lidarr.audio/api/testing/artist/93bd1037-d736-42ca-93c3-c397f2cb1165, though it's fine on other artists. I don't see anything obvious wrong here, but the Sentry stack track only shows the Flask calls. May be worth checking out that artist with your dev instance.

Reply:
Sorry for all the comments, but looks like this is also at least part of the test failures. See https://travis-ci.org/lidarr/LidarrAPI.Metadata/jobs/448969259.